lib/web/apiserver.go

```diff
@@ -1395,9 +1395,17 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		}
 	}
 
+	clusterFeatures, err := h.cfg.ProxyClient.GetFeatures(r.Context())
+
+	if err != nil {
+		h.log.WithError(err).Warn("cannot retrieve cluster features, client may get a stale feature set")
+		// h.ClusterFeatures may be stale, but can be used to fulfill the request
+		clusterFeatures = &h.ClusterFeatures
+	}
+
 	// get tunnel address to display on cloud instances
 	tunnelPublicAddr := ""
-	if h.ClusterFeatures.GetCloud() {
```

Comment:
Does the `h.ClusterFeatures` member is still needed ?

Reply:
I think we should update all `h.ClusterFeatures` calls to `Ping` auth instead, except maybe the ones that only run at startup time.

Or, alternatively, we periodically Ping the auth server and update `h.ClusterFeatures`. I think this may be a good solution to avoid one more network call during user requests.

Reply:
I think that his is worth to consider depending how often the `web/config.js` endpoint is called. 

Since auth  `ping` gRPC endpoint can be called from unauthorised client I think that for now if fine but adding @rosstimothy in the loop.